### PR TITLE
Development

### PR DESCRIPTION
**Pull request by pichar376**
first changes in proyect delivery

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+      </section>
+      <nav>
+        <h1 className="title">Delivery <span className="history">History</span></h1>
+        <section className="search">
+
+          <TextField id="outlined-basic" label="Outlined" variant="outlined" />
+          <Button variant="contained" color="success">New delivery</Button>
+        </section>
+      </nav>
```
Comment (r3ivaj) [inline src/components/Header.js:24]: I think `nav` is not semantic for this case. It should be called `header`

File: src/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+
+
+const Header = () => {
+  return (
+
+    <MyStyleHeader>
+      <section className="dronocargo">
+        <p>Dronocargo</p>
+        <p>Regina Zepeda <span>icon</span></p>
```
Comment (r3ivaj) [inline src/components/Header.js:14]: why is the icon needed? is that a placeholder?

File: src/components/Header.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  );
+}
+
+
+const MyStyleHeader = styled.div`
+display:flex;
+flex-flow:column wrap;
+width:100%;
+border:thin solid red;
+height:10rem;
```
Comment (r3ivaj) [inline src/components/Header.js:39]: I wouldn't use rems unless your font sizes are using rems as well

File: src/components/Input.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import TextField from '@material-ui/core/TextField';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    '& > *': {
+      margin: theme.spacing(1),
+      width: '25ch',
```
Comment (r3ivaj) [inline src/components/Input.js:9]: Why are we using `25ch` as width and not `width: auto` ?

File: src/components/Layout.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+const ContainerLayout = styled.div`
+margin:10rem auto;
+width:60%;
+border:thin solid grey;
+`
+
+
+
+export default Layout
```
Comment (r3ivaj) [inline src/components/Layout.js:25]: please don't forget to add new lines

File: src/components/Layout.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+  return (
+    <ContainerLayout>
+      {children}
+    </ContainerLayout>
+  );
+}
+
+const ContainerLayout = styled.div`
+margin:10rem auto;
+width:60%;
```
Comment (r3ivaj) [inline src/components/Layout.js:19]: this should have a fixed width in pixels, avoid using magic numbers

File: src/components/MenuDropDown.js [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+
+
+
+
+  return (
+    <MydropDownContainer click={click}>
+      <StyledButtonFeatures onClick={handleClick} click={click}>
+        <p></p>
+        <p></p>
+        <p></p>
```
Comment (r3ivaj) [inline src/components/MenuDropDown.js:24]: These bunch of paragraphs are not needed, I would rather use a Box instead https://mui.com/material-ui/react-box/

Or add margins if needed

File: src/components/MenuDropDown.js [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+
+      </StyledButtonFeatures>
+
+
+      <ContainerList click={click} >
+
+        <p className="loguin">loguin or create account</p>
```
Comment (r3ivaj) [inline src/components/MenuDropDown.js:31]: there is a typo here `loguin`, also these project doesn't need an authentication system

File: src/index.css [hunk 9]
```diff
@@ -1,13 +1,22 @@
-body {
+html {
+  box-sizing: border-box;
+  min-width: 320px;
```
Comment (r3ivaj) [inline src/index.css:3]: html selects doesn't need to contain a min width

**Comment by r3ivaj** [comment]
Please start using ESLint in your project, you can integrate ESLint to auto-format your code by pressing control s

**Comment by r3ivaj** [comment]
Also please add the description of your PR, wether you are adding a header, a footer, the table, the search component, etc

File: src/components/Header.js (hunk 1 above)
Comment (pichar376) [reply src/components/Header.js:24]: Of course, you're right, I didn't think it through. I was guided by the search but I'll take it into account in the future.

File: package.json [hunk 10]
```diff
@@ -3,12 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.10.4",
+    "@emotion/styled": "^11.10.4",
```
[the remark this answers is not in the document]
Comment (pichar376) [reply package.json:7]: I installed them by mistake, but I'm going to remove them.

File: src/components/Header.js (hunk 2 above)
Comment (pichar376) [reply src/components/Header.js:14]: In fact if it was a placeholder, in what put the real icon.

File: src/components/Header.js (hunk 3 above)
Comment (pichar376) [reply src/components/Header.js:39]: Entonces es mejor usar pixeles como unidad.o que me recomiedas.

File: src/components/Input.js (hunk 4 above)
Comment (pichar376) [reply src/components/Input.js:9]: well what happens is that this component imports it like this, I have never used inches.

File: src/components/Layout.js (hunk 6 above)
Comment (pichar376) [reply src/components/Layout.js:19]: OK well

File: src/components/MenuDropDown.js (hunk 7 above)
Comment (pichar376) [reply src/components/MenuDropDown.js:24]: Well well.

File: src/components/MenuDropDown.js (hunk 8 above)
Comment (pichar376) [reply src/components/MenuDropDown.js:31]: ok to delete it.haha if i wrote it wrong i forgot to adjust the grammar.

File: src/index.css (hunk 9 above)
Comment (pichar376) [reply src/index.css:3]: ok thanks for all the advice and for your time.

**Pull request merged**